util/testutil/testutil.go

```diff
@@ -324,21 +324,18 @@ type autoRandom struct {
 	originAlterPrimaryKey bool
 }
 
-// SetupAutoRandomTestConfig set alter-primary-key to false, and set allow-auto-random to true and save their origin values.
+// SetupAutoRandomTestConfig set alter-primary-key to false and save its origin values.
 // This method should only be used for the tests in SerialSuite.
 func (a *autoRandom) SetupAutoRandomTestConfig() {
```

Comment:
Do we need to update this function name? This function doesn't handle `AutoRandom` related configurations now.

Reply:
This is only used in `auto_random` tests and it does set up the config. I think no need to update.